test/plots/piePlotTests.ts

```diff
@@ -351,14 +374,14 @@ describe("Plots", () => {
     describe("selections", () => {
       it("retrieves all dataset selections with no args", () => {
         let allSectors = piePlot.selections();
-        assert.strictEqual(allSectors.size(), 2, "all sectors retrieved");
+        assert.strictEqual(allSectors.size(), 2 * 2, "all sectors retrieved");
```

Comment:
Can we assert that the returned `selection` includes the correct number of fill and stroke `<path>`s?